README.rst

```diff
@@ -99,6 +99,34 @@ More information on compilation and development can be found on the
 of the project documentation.
 
 
+Extra
+=====
```

Comment:
The heading level is inconsistent here.  It should format correctly to use this instead:

```rst
Extra
^^^^^

Linux
"""""

Ubuntu
""""""

```